tests/unit_tests/data_steward/validation/main_test.py

```diff
@@ -343,6 +346,30 @@ def test_process_hpo_ignore_dirs(
             self.assertEqual('noob', bucket)
             self.assertTrue(filepath.startswith('SUBMISSION/'))
 
+    @mock.patch('validation.main.run_achilles_export')
+    @mock.patch('gcs_utils.get_achilles_transfer_bucket')
+    def test_run_export_dashboard_bucket(self, mock_transfer_bucket,
+                                         mock_run_achilles_export):
+        """
+        test if the export to achilles dashboard bucket is called with right values
+        :param mock_transfer_bucket: mock achilles transfer bucket name
+        :param mock_run_achilles_export: mock exporting the achilles reports
+        :return: 
+        """
+        expected_bucket_name = 'noob'
+        mock_transfer_bucket.return_value = expected_bucket_name
+        fake_hpo_id = 'fake'
+        datasource_name = fake_hpo_id
+        folder_prefix = fake_hpo_id + '/'
```

Comment:
Might be a good idea to use `os.path.join` and a constant for `datasources.json`